src/Evolution/Executables/ScalarAdvection/EvolveScalarAdvection.hpp

```diff
@@ -313,12 +313,11 @@ struct EvolutionMetavars {
 
       Initialization::Actions::AddComputeTags<
           tmpl::list<ScalarAdvection::Tags::VelocityFieldCompute<Dim>>>,
-      tmpl::conditional_t<
-          is_analytic_solution_v<initial_data>,
-          Initialization::Actions::AddComputeTags<
-              tmpl::list<evolution::Tags::AnalyticCompute<
-                  Dim, initial_data_tag, analytic_solution_fields>>>,
-          tmpl::list<>>,
+      tmpl::conditional_t<is_analytic_solution_v<initial_data>,
+                          Initialization::Actions::AddComputeTags<tmpl::list<
+                              evolution::Tags::AnalyticSolutionsCompute<
+                                  Dim, analytic_solution_fields>>>,
```

Comment:
Is the plan to move this compute tag to the ObservationBox? Maybe add a separate commit to this PR for the executables that support this already?

Reply:
Yes. I don't see a reason to move that code to this PR vs keeping it in the followup where it's actually needed. That's a lot of work for a net-zero effect as far as I can tell.

Reply:
Sure, follow-up is fine. Just checking if I understand the plan correctly.